### DISCUSSION TITLE
Testing setup with MS SQLSERVER

### DISCUSSION THREAD

**Discussion by finchy70** [spatie/laravel-multitenancy#279] [Q&A]
I have a multi database multi tenancy application.
I have been attempting to write php unit tests but can't seem to get a setup that works.

Usually I would use an in memory sqlite db but this does not seem possible with spatie multi tenancy.

I did get tests to sort of work but they were using my dev databases and when using refreshDatabase these were taking a very long time.

I'm using MS SQLSERVER in dev and will be using it in production.

Does anyone have a working setup to run php unit tests?

**Comment by BertKooij** [comment]
I just started a broader discussing related to testing and performance: #320. Although I am not fully satisfied with my current setup I might be able to help. What is the current setup you have (or the setup which you noted to be taking a very long time)?
